kura/examples/scenarios/org.eclipse.kura.example.ai/snapshots/graph-anomaly-detector.xml

```diff
@@ -399,66 +402,6 @@
             </esf:property>
         </esf:properties>
     </esf:configuration>
-    <esf:configuration pid="script">
```

Comment:
we should replace this with the wire.script.tools implementation

Reply:
the test pass without require it

Reply:
ok but it's out of scope of this PR. I'll item an item in the backlog